.github/workflows/ci.yaml

```diff
@@ -148,6 +148,11 @@ jobs:
         charm:
           - jupyter-controller
           - jupyter-ui
+        test-type: [integration, integration-ambient]
+        # TODO: Remove this exclusion once jupyter-controller integration-ambient tests are implemented
+        exclude:
+          - charm: jupyter-controller
+            test-type: integration-ambient
```

Comment:
question: will this eventually be included, when the other charm is made compatible with Ambient?